tests/std/tests/P0024R2_parallel_algorithms_is_partitioned/test.cpp

```diff
@@ -89,9 +118,14 @@ void test_case_is_partitioned_parallel(const size_t testSize) {
 
     // testing with 2 partition adjacent points (T...T F T F...F) where the adjacent partition points are tried at each
     // index
-    first       = c.begin();
-    auto second = next(first, 2);
+    remainingAttempts = quadradic_complexity_case_limit;
+    first             = c.begin();
+    auto second       = next(first, 2);
     for (; second != c.end(); ++first, ++second) {
+        if (--remainingAttempts == 0) {
+            return;

```

Comment:
This is a minor maintenance hazard. Unlike other tests (where the function contains one small loop), this function contains many loops. Having the last loop use `return;` means that if additional test cases were added below (following the existing pattern of the function), they would be silently skipped.

Reply:
I used return here to avoid needing to meet all the postconditions of the loop (iterator positions and so on)

Reply:
I see you changed this to break but that is not correct because the loop mutates the buffer under test...

Reply:
I guess it doesn't matter for now; followup tests might just clear the buffers or something,

Reply:
Whoops, sorry for the mistaken change.